Cargo.toml

```diff
@@ -22,8 +22,8 @@ networking = ["reqwest"]
 name = "Emulsion"
 identifier = "io.github.arturkovacs.emulsion"
 icon = ["resource_dev/emulsion.png"]
-version = "1.9"
 short_description = "A lightweight and minimalistic image viewer"
+copyright = "Copyright (c) 2020 Artúr Barnabás Kovács"
```

Comment:
Wait a second I'm a dummy. The license is also specified in the [package] section above. So what happens if you remove this? What copyright will the built package contain then?

Reply:
No problem, I should have spotted that to. According to cargo-bundle the copyright line is optional but it's not specified whether or not it will use the one from the package section as fallback.

Reply:
I'm assuming that this influences the contenst of the "Info" window. Like this: https://i.stack.imgur.com/28Dqv.png

I'm not experienced with MacOS so I'm shooting in the dark.